src/xrCommon/inlining_macros.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#if defined(__GNUC__)
+#if defined(__GNUC__) || defined(__clang__)
```

Comment:
isn't it compiler wise? Some people try to add support for clang so probably it would be better to add as distinct PR?

Reply:
Yes surely even though it s mainly because I had no much of a choice actually gcc for FreeBSD code generation crashes at the moment ... still to be fixed.